en/docs/concepts/compliance/gdpr.md

```diff
@@ -0,0 +1,226 @@
+-   Anonymize any retained traces of the user activity.  
+    -   Log Files
+    -   Analytics data, related to login, session, key validation, etc.
+    -   Key/token data held at the database layer.
+-   Delete any unwanted data retained in the database(due to performance
+    reasons)
+    -   Token(s) issued,
+    -   Password History information.
+
+For more information on the topic, see [Removing References to Deleted User Identities](TO-DO: insert-extend-link). 
+
+-   **The right for notification obligation -** The **My Account** application can be extended to act as the notification center for
+    individuals.  
+
+-   **The right to data portability -** Individuals can download their
+    user profile in a structured, commonly used and machine-readable
+    JSON document format through the **My Account** application. Ability to download individual’s user profile as a structured, commonly
+used in machine-readable format is a requirement of GDPR. In WSO2 IS,
+it is possible to use one of the following options to download user
+profile as a structured JSON document.
+
+1.  Logging in to the **My Account** application. 
+
+2.  Invoking personal data export API(secure RESTful API)
+
+Additionally, GDPR encourages to facilitate user profile provisioning
+from the data processing organization to another organization automatically based on
+individuals' requests. SCIM2 API supported in WSO2 IS can
+be used to fulfill this requirement.
+
+-   **The right to object -** The **My Account** application can be extended
+    to act as a communication channel to make objections on
+    processing.  
+
+-   **Rights in relation to automated decision making and profiling -**
+    The **My Account** application can be extended to act as a communication
+    channel to make objections on automated decision making and
+    profiling.
+
+The following additional features are also supported in the **My Account** application.
+
+-   Revoking consent for all or specific attributes
+-   Giving an expiry date for a consent
+
+---
+
+## What is consent?
+
+GDPR defines six lawful means of processing individual data. 
+
+- Consent from an individual 
+- Contract with the individual
+- Compliance with a legal obligation
+- Vital interests
+- A public task 
+- Legitimate interests 
+
+Out of these, consent from an individual is considered the most crucial since it is a data processing law that is applicable to a wide spectrum of business activities. For more information on lawful data processing, see [Defining a Winning GDPR Strategy Part 2 - 7 Steps for GDPR Compliance](https://wso2.com/library/article/2017/12/7-steps-for-gdpr-compliance/). 
+
+According to GDPR, the consent is defined as “ *Any freely given,
+specific, informed and unambiguous indication of the data subject’s
+wishes by which he or she, by a statement or by a clear affirmative
+action, signifies agreement to the processing of personal data relating
+to him or her* ”. 
```

Comment:
shall we move this definition to the beginning of the content under this sub heading?

Reply:
I thought it would be better to first let the reader know where the consent management idea is coming from since we haven't explained all the other laws of data processing, rather than jumping right in to the definition. WDYT?